Bottom: 8a663ad426ea58df81a88381b20c2edeed2cac7c
Top:    0b01d29061edd83e718d0288f53d5ecc75644441
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 22:24:31 +0100

Refresh of try-something-with-the-rtrees

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 870500f..8fb26e3 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -529,12 +529,6 @@ cntrbox_adjust (PLINE * c, Vector p)
 
 /* some structures for handling segment intersections using the rtrees */
 
-typedef struct contour
-{
-  BoxType box;
-  PLINE *p;
-} contour;
-
 typedef struct seg
 {
   BoxType box;
@@ -1888,7 +1882,7 @@ poly_AndSubtract_free (POLYAREA * ai, POLYAREA * bi,
       poly_Free (&b);
       assert (poly_Valid (*aminusb));
     }
-  /* delete holes if any left *poly_DelContour */
+  /* delete holes if any left */
   while ((p = holes) != NULL)
     {
       holes = p->next;
